manual/core/native_protocol/README.md

```diff
@@ -24,18 +24,30 @@ only available with specific protocol versions.
 Java driver 4 supports protocol versions 3 to 5. By default, the version is negotiated with the
 first node the driver connects to:
 
-| Cassandra version   | Negotiated protocol version with driver 4 ¹     |
-|---------------------|-------------------------------------------------|
-| 2.1.x (DSE 4.7/4.8) | v3                                              |
-| 2.2.x               | v4                                              |
-| 3.x (DSE 5.0/5.1)   | v4                                              |
-| 4.x ²               | v5                                              |
+| Cassandra version               | Negotiated protocol version with driver 4 ¹     |
+|---------------------------------|-------------------------------------------------|
+| 2.1.x (DSE 4.7/4.8)             | v3                                              |
```

Comment:
Should we remove the DSE versions now that we have a separate table for them?

Reply:
Yes